### ISSUE TITLE
Add policy for vetting guest speakers

### ISSUE THREAD

**Issue by kburk1997** [rcos/rcos-handbook#90]
**Description of Issue:**

Currently, we are very lenient towards allowing guest speakers. There has been controversy in our community on whether or not to allow guest speakers from private companies to give talks on closed-source software.

**Proposed Solution:**

Document a simple vetting process for guest speakers. A draft policy is asking the following questions:
* What is your talk about?
* How is it related to open source / CS / design / engineering?
* Do you agree to abide by the RCOS Community Code of Conduct in your talk>

**Comment by vakrao** [comment]
Yeah, I think maybe having external speakers like provide their slides 24-48hrs in advance would be helpful as well to vet them.
